### ISSUE TITLE
Error while trying to enterScope with Sf3

### ISSUE THREAD

**Issue by ibraelillo** [FriendsOfSymfony/FOSRestBundle#1293]
hello, 

I'm describing my routes as follow 

```yaml
users:
    resource: "app.controller.user_controller"
    prefix: /
    type: rest
```

and my service "app.controller.user_controller" 
   
```yaml
app.controller.user_controller:
        parent: rcu_api.controller.api
        class: AppBundle\Controller\UserController
        calls:
            - [ setRepository, ["@tux.repository.user"]]
```

The same controller service configuration works well in 2.7 as I have it in production right now, but whithin the new app in sf3, I'm having an error while tring to load routes:

**UndefinedMethodException in RestRouteLoader.php line 120:
Attempted to call an undefined method named "enterScope" of class "appDevDebugProjectContainer".**

the image shows the error.
![error sf3](https://cloud.githubusercontent.com/assets/3011753/12034553/c5308c74-ae32-11e5-8a2c-249fe8bcadce.png)

while I'm able to access to controller using class path, I cant use my service as a controller, and dependency injection don't work as expected in 2.7.

Maybe I'm missing something here? after my inspection of cache clase appDevDebugProjectContainer  i'm realizing that this class has not an "enterScope" method. 

It's there a workaround?

**Comment by Ener-Getick** [comment]
It seems that we forgot a fix in ``1.7``... I'll fix it this afternoon.

**Comment by xabbuh** [comment]
We have a check using `method_exists()`: https://github.com/FriendsOfSymfony/FOSRestBundle/blob/master/Routing/Loader/RestRouteLoader.php#L120

@ibraelillo Which version of the bundle do you use?

**Comment by Ener-Getick** [comment]
Not in 1.7 @xabbuh

**Comment by xabbuh** [comment]
Indeed, maybe we can simply backport that patch.

**Comment by Ener-Getick** [comment]
I released a new version including #1294, thank you @ibraelillo :-)
